helpers.go

```diff
@@ -640,3 +640,24 @@ func NewCallbackWithAlert(id, text string) CallbackConfig {
 		ShowAlert:       true,
 	}
 }
+
+func NewInvoice(chatID int64, title, description, payload, providerToken, startParameter, currency string, prices *[]LabeledPrice, photoUrl string, photoSize, photoWidth, photoHeight int, needName, needPhoneNumber, needEmail, needShippingAddress, isFlexible bool) InvoiceConfig {
```

Comment:
Only required fields should be passed to it, anything not required should just be set on the config afterwards. All the optional fields should just be set to sane defaults.